pygotham/talks/forms.py

```diff
@@ -1,16 +1,22 @@
 """Talks forms."""
 
 from flask.ext.wtf import Form
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
 from wtforms.validators import Optional
 from wtforms_alchemy import model_form_factory
 
-from pygotham.talks.models import Talk
+from pygotham.talks.models import Duration, Talk
 
 __all__ = ('TalkSubmissionForm',)
 
 ModelForm = model_form_factory(Form)
 
 
+def duration_query_factory():
+    """Return available :class:`~pygotha.models.Duration` instances."""
+    return Duration.query.filter(Duration.inactive == False).all()
```

Comment:
Probably. All the examples I've ever seen include `all`, but it doesn't appear to be required.